pkg/apis/serving/v1alpha2/kfservice_types.go

```diff
@@ -94,11 +94,11 @@ type TransformerSpec struct {
 type AlibiExplainerType string
 
 const (
-	AlibiAnchorsTabularExplainer  AlibiExplainerType = "AnchorsTabular"
-	AlibiAnchorsImageExplainer    AlibiExplainerType = "AnchorsImage"
-	AlibiAnchorsTextExplainer     AlibiExplainerType = "AnchorsText"
-	AlibiCounterfactualsExplainer AlibiExplainerType = "Counterfactuals"
-	AlibiContrastiveExplainer     AlibiExplainerType = "Contrastive"
+	AlibiAnchorsTabularExplainer  AlibiExplainerType = "anchor_tabular"
```

Comment:
These are exposed via the API right? So we'd have snake case in the yaml?

Reply:
Yes- so I should change all to snake_case. They are passed directly as command line args to the explainer image.

Reply:
Updated.